### PR TITLE
[TDV-561] Redesign status message

### PR DESCRIPTION
**Pull request by ValeriaNARDET**

### CODE REVIEW

File: src/vue/common/StatusMessage.vue [hunk 1]
```diff
@@ -91,53 +104,130 @@ export default {
...
+    padding: 2rem 2rem 0.5rem;
+
+    p {
+      font-size: 1.8rem;
+    }
+  }
+
+.status-message__text {
+  margin: 2rem;
 }
```
Comment (max-nes) [inline src/vue/common/StatusMessage.vue]: this rule could be merged with status-message__content

File: src/vue/common/StatusMessage.vue [hunk 2]
```diff
@@ -91,53 +104,130 @@ export default {
...
   @include respond-to-custom($status-message-reposition-bp) {
     right: $content-side-paddings-sm;
     left: $content-side-paddings-sm;
-    padding: 2rem;
+    opacity: 1;
   }
 
-  @mixin apply-theme ($col-msg-background, $col-msg-text) {
-    background: $col-msg-background;
+  @mixin apply-theme ($col-bg, $col-bg-secondary, $col-msg) {
```
Comment (max-nes) [inline src/vue/common/StatusMessage.vue]: try define only one apply-theme for all the usages

File: src/vue/common/StatusMessage.vue [hunk 3]
```diff
@@ -91,53 +104,130 @@ export default {
...
+    @include apply-theme($col-error,$col-error-secondary,  $col-msg-error)
+  }
+  &--info {
+    @include apply-theme($col-info,$col-info-secondary,  $col-msg-info)
+  }
 }
 
-.status-message__content {
-  margin-bottom: 2.5rem;
+  .status-message__content {
```
Comment (max-nes) [inline src/vue/common/StatusMessage.vue]: pls keep code formatting clean

File: src/vue/common/StatusMessage.vue [hunk 4]
```diff
@@ -1,23 +1,36 @@
...
-          "status-message.close-lbl" | globalize({
-            context: messageType
-          })
-        }}
-      </button>
+  <transition name="toggle">
+    <div
+      v-if="isShown"
+      :class="`status-message status-message--${messageType}`"
+    >
```
Comment (max-nes) [inline src/vue/common/StatusMessage.vue]: the message still looks a bit not cool. please try again

**Comment by ValeriaNARDET** [comment]
need to add test for status message title

**Pull request merged**